model/task_lifecycle_test.go

```diff
@@ -5260,6 +5260,11 @@ func TestResetStaleTask(t *testing.T) {
 			require.NoError(t, err)
 			require.NotZero(t, dbVersion)
 			assert.Equal(t, evergreen.VersionCreated, dbVersion.Status, "version status should be updated for restarted task")
+
+			dependencyTask, err := task.FindOne(db.Query(task.ById("dependencyTask")))
```

Comment:
Also have to `require.NotNil(t, dependencyTask)` or else L5266 can panic.

Same with the check below this.